Pipfile

```diff
@@ -8,10 +8,12 @@ coniql = {editable = true, extras = ["dev"], path = "."}
 
 [packages]
 coniql = {editable = true, path = "."}
+pytest-aiohttp = "*"
 
 [scripts]
 lint = "pre-commit run --all-files --show-diff-on-failure --color=always -v"
-tests = "pytest"
+# Not running tests until full subscription/query/put are implemented
+# tests = "pytest"
```

Comment:
The only reason I hadn't put the automated tests back in was because there was still a test for the pvacces plugin which has not been updated as we deemed this not a current priority. However I agree that the other tests are now ready to run and so I have removed the pvaccess tests as we can add these when we add the support for pvaccess. See commit: [a14ace5](https://github.com/dls-controls/coniql/pull/44/commits/a14ace5f57f893fa844479d1e61bdf390ef6f650).